sonar-python-plugin/src/main/java/org/sonar/plugins/python/parser/StaxParser.java

```diff
@@ -42,32 +38,13 @@
 
   private XmlStreamHandler streamHandler;
 
-  private boolean isoControlCharsAwareParser;
-
-  /**
-   * Stax parser for a given stream handler and iso control chars set awareness to "off".
-   */
   public StaxParser(XmlStreamHandler streamHandler) {
-    this(streamHandler, false);
-  }
-
-  /**
-   * Stax parser for a given stream handler and iso control chars set awarness to "on".
-   * The iso control chars in the xml file will be replaced by simple spaces, usefull for
-   * potentially bogus XML files to parse, this has a small perfs overhead so use it only when necessary.
-   */
-  public StaxParser(XmlStreamHandler streamHandler, boolean isoControlCharsAwareParser) {
     this.streamHandler = streamHandler;
-    XMLInputFactory xmlFactory = XMLInputFactory.newInstance();
-    if (xmlFactory instanceof WstxInputFactory) {
-      WstxInputFactory wstxInputfactory = (WstxInputFactory) xmlFactory;
-      wstxInputfactory.configureForLowMemUsage();
-      wstxInputfactory.getConfig().setUndeclaredEntityResolver(new UndeclaredEntitiesXMLResolver());
```

Comment:
@ivandalbosco I think that we should not remove this line: we now fail if we find `&xxx;` in the parsed XML.